### DISCUSSION TITLE
ENOENT: no such file or directory, open './secure-connect-akijturbo.zip'

### DISCUSSION THREAD

**Discussion by mohiandipta** [vercel/community#2469] [Help]
### Summary

after deploying into the vercel giving this error:
```
[Error: ENOENT: no such file or directory, open '.src/config/secure-connect-akijturbo.zip'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: './secure-connect-akijturbo.zip'
}
Error: ENOENT: no such file or directory, open '.src/config/secure-connect-akijturbo.zip'
```

but everything is working properly in locally.

### Additional information

in my code i'm actually using node.js and cassandra. and the zip file is for connection bundle for datastax cassandra with node.js. 

```
const connection: Connection = {
        client: new Client({
            cloud: { secureConnectBundle: process.env.SECURE_CONNECT_BUNDLE as string },
            keyspace: keyspace,
            credentials: { username: USERNAME!, password: PASSWORD! },
        }),
    };
```
Here in cloud `secureBundleConnection` getting zip file path from `.env` like this
```
SECURE_CONNECT_BUNDLE= "./src/config/secure-connect-akijturbo.zip"
```

My `vercel.json` configuration
```
{
  "version": 2,
  "name": "akij-server",
  "builds": [{ "src": "index.ts", "use": "@vercel/node" }],
  "routes": [{ "src": "/(.*)", "dest": "/index.ts" }]
}
```

**Comment by amyegan** [comment]
Do you have a minimal reproduction that we can use to get the full picture of what is going wrong?

**Reply by mohiandipta** [reply]
are you talking about deployment preview?

by the way, i had to move from cassandra to mongodb because it was emergency for my production. I think that was the issue of reading zip file from my node.js application. But if so then i'm not sure how it can read zip file while in locally running

**Comment by LModipane** [comment]
@mohiandipta, did you manage to resolve your issue.

**Reply by mohiandipta** [reply]
i didn't. it was emergency so i moved to another database for continue my work.
